### PR TITLE
All the fails

### PR DESCRIPTION
**Pull request by codestreamtest**
Don't even know where to begin.

### CODE REVIEW

File: inventory.js [hunk 1]
```diff
@@ -21,6 +21,9 @@ import { Container } from "../container";
 import { Logger } from "../logger";
 import { Foo } from "../foo";
 
+// this is on feature/fail
+// so that I can test moar PRs
```
Comment (planteater) [inline inventory.js:25]: Review the codz

**Review by planteater — APPROVED** [review]
awesomeness

**Review by codestreamtest2 — APPROVED** [review]
Whatev

**Comment by codestreamtest** [comment]
how we looking?

File: inventory.js (hunk 1 above)
Comment (codestreamtest) [inline inventory.js:25]: fix the things

**Review by codestreamtest — COMMENTED** [review]
ok whatev

**Comment by planteater** [comment]
Help here

File: inventory.js [hunk 2]
```diff
@@ -21,6 +21,9 @@ import { Container } from "../container";
...
+
```
Comment (planteater) [inline inventory.js:26]: looks wonderful!

**Review by planteater — APPROVED** [review]
Looks great!

**Comment by planteater** [comment]
not part of changeset

```
	export function findEditor(uri: Uri, lastActive?: TextEditor): TextEditor | undefined {

```
								
[inventory.js](https://github.com/codestreamtest/beerfridge/blob/59f506bfc303ef1200042303085bd1cc529a6396/inventory.js#L42) (Line 42)

**Comment by planteater** [comment]
hi github

File: inventory.js (hunk 2 above)
Comment (planteater) [inline inventory.js:26]: look super

**Review by planteater — APPROVED** [review]
awesome

**Comment by planteater** [comment]
this is from CS

File: inventory.js [hunk 3]
```diff
@@ -21,6 +21,9 @@ import { Container } from "../container";
 import { Logger } from "../logger";
 import { Foo } from "../foo";
 
+// this is on feature/fail
```
Comment (planteater) [inline inventory.js:24]: this is a pr comment

**Review by planteater — APPROVED** [review]
Approved!

**Pull request merged**